src/components/SliderWithInput.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Slider, InputNumber,
+} from 'antd';
+
+import useUpdateThrottled from '../utils/customHooks/useUpdateThrottled';
+
+const SliderWithInput = (props) => {
+  const {
+    min, max, value, onUpdate, disabled, step,
+  } = props;
+
+  const [, handleChange] = useUpdateThrottled(onUpdate, value);
+
+  const [newLocalValue, setNewValue] = useState(value);
+
+  useEffect(() => {
```

Comment:
Out of curiosity: is this needed? Isn't that value of `value` already stored in the state?